src/Tizen.NUI.Components/Controls/SelectGroup.cs

```diff
@@ -69,6 +69,12 @@ public abstract class SelectGroup
         [EditorBrowsable(EditorBrowsableState.Never)]
         public int SelectedIndex => selectedIndex;
 
+        /// <summary>
+        /// EnableMultiSelected is used to indicate if SelectGroup can select multiple SelectButtons.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool EnableMultiSelected { get; set; } = true;
```

Comment:
How about EnableMultiSelection?

Reply:
Thank you for your opinion!
I thought that "Selection" is used for Text selection so it may confuse users.
However, MultiSelection sounds much more natural.

BTW, based on your opinion, I think SelectedChanged and SelectedChangedEventArgs may be required to be changed their names to SelectionChanged and SelectionChangedEventArgs.
This can be done in API9 by deprecating the SelectedChanged and SelectedChangedEventArgs.

I have updated the code.
Thank you.